go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -1162,3 +1192,7 @@ func (bnc *BaseNetworkController) wasPodReleasedBeforeStartup(uid, nad string) b
 	}
 	return bnc.releasedPodsBeforeStartup[nad].Has(uid)
 }
+
+func isAllowedForMigration(isSecondary, isPrimaryNetwork, isl2Topology bool) bool {
+	return isSecondary && !isPrimaryNetwork && isl2Topology
```

Comment:
so we don't allow migration for the primary network ? That defeats the purpose of the PR - we want to allow this both for primary and secondary UDN.

Reply:
At the time this PR was meant to address only secondary UDNs, but it was meant to change when we had the new VM with managedTap.

Reply:
This can be resolved, right ?